src/app/ubs/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html

```diff
@@ -27,7 +27,15 @@ <h3 class="form__headline">{{ 'customer-info.title' | translate }}</h3>
             <label class="form__label" for="customer-tel">
               {{ 'customer-info.customer.tel-label' | translate }}
             </label>
-            <input class="form-control" type="tel" id="customer-tel" placeholder="{{ userInfo.customerPhoneNumber }}" disabled />
+            <input
+              class="form-control"
+              type="tel"
+              id="customer-tel"
+              placeholder="{{
+                userInfo.customerPhoneNumber.charAt(0) !== '+' ? '+' + userInfo.customerPhoneNumber : userInfo.customerPhoneNumber
```

Comment:
try `'+' + userInfo.customerPhoneNumber.replace(/^\+/, '');`
it would be simpler I guess